worker/Makefile

```diff
@@ -225,6 +225,11 @@ else
 	$(BUILD_DIR)/mediasoup-worker-test --invisibles --use-colour=yes $(MEDIASOUP_TEST_TAGS)
 endif
 
+test-asan: setup
```

Comment:
You must add "test-asan" also in a list with all task names at the top of this file.